src/components/model.jsx

```diff
@@ -134,7 +134,10 @@ export default function Model({ app, appLayout, isLocalStorage }) {
         target={`[data-boxid="${currentDetailsView.boxId}"]`}
         content={currentDetailsView.content}
         callback={(event) => { if (event === 'close') { setCurrentDetailsView(null); } }}
-      />
+      >
+        <>
```

Comment:
❓

Reply:
For some reason the Floater component has children as required prop.. and the children is supposed to be the element that triggers the floater to open. We use the autoOpen prop which opens the floater on e.g. a state, and don't want any children. So long story short, the `<>`(the new `<React.Fragment>`) is to get rid of a warning.